### ISSUE TITLE
Unremovable bottom padding when using line-height

### ISSUE THREAD

**Issue by keturiosakys** [raphamorim/rio#449]
I've set the line height property to `1.2` and it seems that all terminal rendering now has this bottom padding (see screenshot):

```toml
line-height = 1.2
```

<img width="2628" alt="CleanShot 2024-02-05 at 17 24 12@2x" src="https://github.com/raphamorim/rio/assets/17225208/6d87e99e-727b-4ebf-b782-e3aa40d00183">

**Comment by Artemis21** [comment]
I have a similar issue. For me, the padding appears above the tab list (with `navigation.mode = "BottomTab").

![image](https://github.com/raphamorim/rio/assets/57376638/5c8d589d-8fd6-4aa5-b871-1b73075059f3)

Full config:
<details>
<summary>Full config</summary>
```toml

**Comment by taggon** [comment]
Same here. Also the viewport overflows when you set the line height to a value less than 1.

**Comment by raphamorim** [comment]
Thanks for the issue!

Yea, currently line-height property is quite unstable, I am fixing it in #428 (which will add some text rendering features as well).

**Comment by ehamiter** [comment]
Hi @raphamorim -- looks like #428 was merged but seems the padding is still an issue?

**Comment by raphamorim** [comment]
Yes, it is still an issue. Will be looking it for the upcoming versions (probably next one), sorry about delay!

**Comment by kristoferlund** [comment]
@raphamorim Just started using Rio the other day, feels like a great terminal! The ability to adjust the line height is the only thing I am missing so far.

**Comment by raphamorim** [comment]
Ok, fyi: I will finally look on this for v0.2.0 (next version)

**Comment by doongjohn** [comment]
The line-height option is not working in 0.2.1. Is the value fixed to 1.0 for now?

**Comment by raphamorim** [comment]
No 😢 sorry peeps, I had to cut release earlier. I am looking this and https://github.com/raphamorim/rio/issues/124

**Comment by mantacid** [comment]
this issue can be triggered by certain fonts, even when using the default value for line height. Rio's default font doesn't trigger this behavior, but a font like unscii will cause this same issue.

**Comment by raphamorim** [comment]
as many folks requested, line height support is coming to next version of rio terminal

![line_height](https://github.com/user-attachments/assets/88a19662-e2b7-4cb7-b093-1f27f1c48f4d)

**Comment by raphamorim** [comment]
<img width="912" alt="Screenshot 2025-01-06 at 11 08 02" src="https://github.com/user-attachments/assets/4448d54f-aaee-4fab-8741-0edc71fb9882" />
